packages/chopsticks/src/cli.ts

```diff
@@ -81,6 +82,27 @@ yargs(hideBin(process.argv))
       await runBlock(await processArgv(argv))
     }
   )
+  .command(
+    'try-runtime',
+    'Runs runtime upgrade',
+    (yargs) =>
+      yargs.options({
+        ...defaultOptions,
+        'output-path': {
```

Comment:
add it here please
```suggestion
'wasm-override': {
    desc: '...',
    string: true,
    required: true
  },
        'output-path': {
```

Reply:
so it will not check override in config? so we have 2 places of checks. is 2 checks okey?

Reply:
I mean remove check from tryRuntime function and add it here. `wasm-override is` defined on `...defaultOptions` but it's optional. we are making it required for try-runtime command

Reply:
Having it on cli command will give you the benefit of help `yarn start try-runtime --help`

Reply:
@dzmitry-lahoda can you do this last change so we can merge?